predictionnet/utils/huggingface.py

```diff
@@ -0,0 +1,66 @@
+import os
+from typing import List
+
+from huggingface_hub import HfApi, model_info
+
+from predictionnet.protocol import Challenge
+
+
+class HF_interface:
```

Comment:
Can we remove underscores from class names? Comparable to `HfAPI` import, maybe we name the class `HfInterface`